test/arquillian-extension/src/main/java/io/narayana/lra/arquillian/spi/NarayanaLRARecovery.java

```diff
@@ -37,27 +57,43 @@ public boolean waitForEndPhaseReplay(URI lraId) {
     }
 
     /**
-     * Invokes LRA coordinator recovery REST endpoint and returns whether the recovery of intended LRAs happened
+     * Invokes LRA coordinator recovery REST endpoint and returns whether the
+     * recovery of intended LRAs happened
      *
      * @param lraId the LRA id of the LRA that is intended to be recovered
      * @return true the intended LRA recovered, false otherwise
      */
     private boolean recoverLRAs(URI lraId) {
         // trigger a recovery scan
-
         try (Client recoveryCoordinatorClient = ClientBuilder.newClient()) {
             URI lraCoordinatorUri = LRAConstants.getLRACoordinatorUrl(lraId);
-            URI recoveryCoordinatorUri = UriBuilder.fromUri(lraCoordinatorUri)
-                    .path(RECOVERY_COORDINATOR_PATH_NAME).build();
+            URI recoveryCoordinatorUri = UriBuilder.fromUri(lraCoordinatorUri).path(RECOVERY_COORDINATOR_PATH_NAME)
+                    .build();
```

Comment:
It looks like there are some formatting changes only in this method and it's documentation that could possibly be reverted?

Reply:
Agreed, I have updated the PR, thanks